src/frontend/src/components/public/PIANavButton/index.tsx

```diff
@@ -49,8 +49,7 @@ const PIANavButton = ({ pages }: INavButton) => {
         )}
         {HandleState(pages, pathname, 'next') && (
           <button
-            type="submit"
```

Comment:
See if you need to add `type="button"`; In my knowledge, submit is considered default

Reply:
cool. Last week we had some weird behaviour accessing buttons without the "button" type.  If you have verified that too, I am good 👍